cleanlab_studio/studio/trustworthy_llm.py

```diff
@@ -0,0 +1,21 @@
+from typing import Tuple
+from cleanlab_studio.internal.api import api
+
+
+class TLM:
+    """Handler for interactions with Trustworthy LLMs."""
+
+    def __init__(self, api_key: str) -> None:
+        """Initializes Trustworthy LLM hanlder w/ API key."""
+        self._api_key = api_key
+
+    def prompt(self, input: str) -> Tuple[str, float]:
```

Comment:
Should we change the return type to something more easily expandable in a compatible way? Right now, if we add more scores, we'd need to add a separate function or break compatibility.

Some ideas:

- Return a dict, with keys `answer` and `confidence_score`
- Return a tuple of `answer, metadata`, where metadata is a dict (and currently only contains `confidence_score`)

Reply:
Remember that if we do this, all docs need to be updated

Reply:
agreed that we should change to something more expandable. I'll do the prior (dict w/ answer and confidence_score), since I think both are first-class return values